agir/lib/display.py

```diff
@@ -46,6 +47,28 @@ def display_price(price, price_in_cents=True):
     return "{}\u00A0€".format(floatformat(price, 2))
 
 
+def display_allocations(obj):
+    allocations = json.loads(obj)
```

Comment:
Pour le rendre plus générique on peut passer directement l'instance en argument :
```suggestion
def display_allocations(instance):
    allocations = instance.meta.get("allocations", "{}")
    allocations = json.loads(allocations)
```